### DISCUSSION TITLE
Assegnarsi i capitoli da elaborare e pubblicare

### DISCUSSION THREAD

**Discussion by aborruso** [ondata/guidaPraticaPubblicazioneCSV#52] [Idee/Proposte]
Buongiorno,
ho creato una issue, per ogni capitolo ancora da trattare. Alcune note:

- così facendo, ognuno potrà vedere quali sono i capitoli orfani, dalla [lista delle issue con etichetta `da elaborare` e non assegnate a nessuno](https://github.com/ondata/guidaPraticaPubblicazioneCSV/issues?q=is%3Aissue+is%3Aopen+label%3A%22da+elaborare%22+no%3Aassignee+sort%3Acreated-asc);
- in ogni issue, ci sono dei task di base da verificare;
- ognuno se la potrà autoassegnare;
- qui alcune [note di lavoro](https://ondata.github.io/guidaPraticaPubblicazioneCSV/note_redazione/).

![image](https://user-images.githubusercontent.com/30607/147463397-37af66cc-7b93-4dc6-92b8-42ff867fb625.png)

Buon lavoro

**Comment by dennisangemi** [comment]
Ciao @aborruso , mi confermi che si potrebbe utilizzare la [pagina](https://gbvitrano.github.io/guidaPraticaPubblicazioneCSV-ES/) preparata da  @gbvitrano per fare tradurre Google translate (o estensione analoga) - tanto per avere una base da cui partire - e infine fare tutti i check del caso presenti nella issue? 

P.s.
Non appena potrò, vi darò una mano perché penso sia un progetto molto molto importante (in primis per me)

**Reply by aborruso** [reply]
> Ciao @aborruso , mi confermi che si potrebbe utilizzare la [pagina](https://gbvitrano.github.io/guidaPraticaPubblicazioneCSV-ES/) preparata da @gbvitrano per fare tradurre Google translate (o estensione analoga) - tanto per avere una base da cui partire - e infine fare tutti i check del caso presenti nella issue?

Ciao caro, [qui](https://github.com/ondata/guidaPraticaPubblicazioneCSV/tree/main/risorse/elaborazioni) trovi alcune traduzioni automatiche. Puoi usare anche quella di Gianni.

> P.s. Non appena potrò, vi darò una mano perché penso sia un progetto molto molto importante (in primis per me)

Mitico!

**Reply by dennisangemi** [reply]
> qui trovi alcune traduzioni automatiche

Grazie mille! Dovevo immaginare esistessero già ;)
